linux_os/guide/system/bootloader-grub2/grub2_pti_argument/rule.yml

```diff
@@ -0,0 +1,56 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Enable Kernel page-table isolation'
+
+description: |-
+    To enable Kernel page-table isolation,
+    add the argument <tt>pti=on</tt> to the default
+    GRUB 2 command line for the Linux operating system in
+    <tt>/etc/default/grub</tt>, in the manner below:
+    <pre>GRUB_CMDLINE_LINUX="pti=on"</pre>
+
+rationale: |-
+    Kernel page-table isolation is a kernel feature that mitigates
+    the Meltdown security vulnerability and hardens the kernel
+    against attempts to bypass kernel address space layout
+    randomization (KASLR).
+
+severity: medium
```

Comment:
For discussion: Given the severity of Meltdown, should this be ``high``?

Reply:
I have no opinion. Updated in eba83009dcecfdba01d9b46668431f49bc1ffc40.